### DISCUSSION TITLE
Join the GitHub Copilot waitlist

### DISCUSSION THREAD

**Discussion by darverise** [community/community#17113] [Other Feature Feedback, Questions, & Ideas]
hello,github

Request to Join the GitHub Copilot waitlist.

Please, please give me a mandate.

**Comment by divyadhaipulle09** [comment]
hello,github

Request to Join the GitHub Copilot waitlist.

Please, please give me a mandate.

**Comment by tianchengdemo** [comment]
hello,github

Request to Join the GitHub Copilot waitlist.

Please, please give me a mandate.

**Comment by zhaifanhua** [comment]
hello,github

Request to Join the GitHub Copilot waitlist.

Please, please give me a mandate.

**Comment by jabranemohamed** [comment]
hello,github

Request to Join the GitHub Copilot waitlist.

I want to increase suggestions related to Spring, Spring data,Spring security etc , java (lambda etc) 

Please, give me a mandate.
